Modules/Accelerometer/accelerometer_factory.py

```diff
@@ -0,0 +1,14 @@
+from . import config
+
+if config.ACCELEROMETER_CONNECTED:
+    from . import mpu9250
+else:
+    from . import fake_accelerometer
+
+#this function returns the mpu9250 hardware module object if the hardware is connected.
+#otherwise returns and object that simulates hardware
+def initialize():
+    if(config.ACCELEROMETER_CONNECTED == True):
```

Comment:
No, you do not do it like that.

It's better to make two hidden functions, for example, `_init_mpu` and `_init_fake`. Then just assign `initialize = _init_mpu if config.ACCELEROMETER_CONNECTED else _init_fake`.

Reply:
The problem is i can't write these function before the import statements and i can't assign ```initialize``` to any of the methods before creating the methods.  
I wrote some code in 2c04131 but i don't think it's optimal.

Reply:
You're right, there are better ways to write it. However, it's still better than checking every time. You can leave it like that.